src/classes/BaseMap.ts

```diff
@@ -42,8 +44,10 @@ export class BaseMap {
 
     /* Instantiate new controls with custom event handlers */
     const changeLayers = new MapboxGLButtonControl("mapbox-gl-change_layer icon icon-layers", "Change Layer", nextLayer, "")
+    const viewResetter = new MapboxGLButtonControl("mapbox-gl-change_layer icon icon-view", "Reset view", this.resetView.bind(this), "")
 
     /* Add Controls to the Map */
+    this.map.addControl(viewResetter, "top-right")
```

Comment:
In the mockup the order of viewResetter and changeLayers is not the same, they are also in the same button group.

It is details, it is not necessary to change it.

Reply:
Oui, on va le garder en tête pour la V2, la j'ai repris l'existant